src/predict/jobs/train/Dockerfile

```diff
@@ -7,9 +7,17 @@ COPY . /app/
 WORKDIR /app/
```

Comment:
Just noticing this new directory for predict-job......care to share some background thoughts on this? @danielogen